esmvaltool/recipes/recipe_collins13ipcc.yml

```
@@ -0,0 +1,3071 @@
+# recipe_collins13ipcc.yml
+
+---
+documentation:
+  description: |
+    Recipe reproducing selected figures from IPCC AR5,
+    chap. 12 (Collins et al., 2013) mainly difference maps between
+    future and present (no observations included), timeseries,
+    changes scaled by mean global temperature change.
+    Only a subset of models are included for most examples!
+    Diagnostic IAV_calc_thetao (zonal ocean temperature, variable thetao) needs
+    a lot of memory, make sure you have enough memory available to read
+    piControl runs or exclude! 
+
+  authors:
+    - lore_ru
```

```suggestion
    - lorenz_ruth
```
Authors' naming convention has changed (see #1164)